borrowing-module/vaults/settlement.md

```diff
@@ -0,0 +1,25 @@
+---
+description: Settlement - Stopping all operations for a specific vault type
+---
+
+# Vault Settlement
+
+As in the Core module, the Borrowing module has a settlement mechanism to stop unsustainable vault types and pay back users. Its goal is to ensure that agTokens issued through this module are **always backed by collateral**. 
+
+## Idea
+The vault settlement should be used only in extreme cases, for example when vaults from a specific [`VaultManager`](/borrowing-module/glossary.md) contract start to be under-collateralized and liquidator stop liquidating unhealthy positions. 
```

Comment:
Need to link to Maker global settlement and to the fact that by liquidating all vault contracts at the same time, you can technically put an end to a stablecoin

Reply:
Also need to insist that it's the place where stablecoins can be redeemed at oracle value, meaning that it's part of what makes agTokens ultimately stable

Reply:
Some "éléments de langage" can be taken from here: https://blog.makerdao.com/introduction-to-emergency-shutdown-in-multi-collateral-dai/

Reply:
For the 1st point, it's not the case for us as there is the other module
We would have to do settlements in both modules at the same time for that

Reply:
Yes so to the module